Lib/test/test_capi/test_misc.py

```diff
@@ -1098,6 +1098,55 @@ class Data(_testcapi.ObjExtraData):
         del d.extra
         self.assertIsNone(d.extra)
 
+    def test_sys_getattr(self):
+        sys_getattr = _testcapi.sys_getattr
+
+        self.assertIs(sys_getattr('stdout'), sys.stdout)
+        with support.swap_attr(sys, '\U0001f40d', 42):
+            self.assertEqual(sys_getattr('\U0001f40d'), 42)
+
+        with self.assertRaisesRegex(RuntimeError, r'lost sys\.nonexisting'):
```

Comment:
This error message is surprising. sys has no attribute "nonexisting". It's not "lost", it simply doesn't exist.

I would prefer to always raise AttributeError with a message like "module 'sys' has no attribute 'x'", similar than in Python:

```
>>> import sys
>>> sys.x
AttributeError: module 'sys' has no attribute 'x'
```